python/ray/serve/pipeline/deployment_node.py

```diff
@@ -85,8 +85,15 @@ def replace_with_handle(node):
                 ),
                 init_args=replaced_deployment_init_args,
                 init_kwargs=replaced_deployment_init_kwargs,
+                route_prefix=original_deployment.route_prefix,
             )
+            # TODO (jiaodong): Unify this with other deployment configs
+            self._bound_other_args_to_resolve[
+                "route_prefix"
+            ] = original_deployment.route_prefix
```

Comment:
do we still need this

Reply:
nope im cleaning them up now, latest one is simplest